hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -1623,35 +1615,6 @@ public void testGetTrashRoots() throws IOException {
     Assert.assertEquals(6, res.size());
   }
 
-  /**
-   * Check that files are moved to trash.
-   * since fs.rename(src,dst,options) is enabled.
-   */
-  @Test
-  @Flaky("HDDS-6646")
-  public void testRenameToTrashEnabled() throws Exception {
```

Comment:
Would you update the PR description to reflect that `testRenameToTrashEnabled` is removed for now?

Reply:
+1

Reply:
@smengcl @hemantk-12 , Thanks for noticing that. Updated the PR description.